chart/templates/configmap-env.yaml

```diff
@@ -31,8 +35,16 @@ data:
   MALLOC_ARENA_MAX: "2"
   NODE_ENV: "production"
   RAILS_ENV: "production"
+  {{- if .Values.redis.enabled }}
   REDIS_HOST: {{ template "mastodon.redis.fullname" . }}-master
+  {{- else }}
+  REDIS_HOST: {{ .Values.redis.hostname }}
+  {{- end }}
+  {{- if .Values.redis.port }}
+  REDIS_PORT: {{ .Values.redis.port }}
```

Comment:
same here, this would benefit from `default` + `quote`

Reply:
thanks for the hint